framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/OpenShiftDeployment.java

```diff
@@ -88,9 +88,13 @@ public void deleteInstances(List<Instance> instances) {
 
     public abstract String getServiceName();
 
+    public String getDeploymentConfigName() {
```

Comment:
Shouldn't be better to have this method abstract too, so for every Deployment will be this set as needed? As you mentioned in comment of this PR, we need this for AMQ and in this PR I don't see change form AMQ deployment impl.

Reply:
The AMQ usage will come in next PR. I try to keep PRs as small as possible to amke reviewes easier.
The reason of not making this method abstract is that for most deployments their deployment config name is equal to service name. Making this method abstract would force all deployments to implement this method.This approach allows deployment to override method only if they need to define custom deployment config name.